ecc_go/ccToolsDemoEnclave.json

```diff
@@ -0,0 +1,33 @@
+{
+ "exe": "ecc",
+ "key": "private.pem",
+ "debug": true,
+ "heapSize": 512,
+ "productID": 1,
+ "securityVersion": 1,
+ "mounts": null,
+ "files": null,
+ "env": [
+    {
+        "name": "CHAINCODE_SERVER_ADDRESS",
+        "fromHost": true
+    },
+    {
+        "name": "CHAINCODE_PKG_ID",
+        "fromHost": true
+    },
+    {
+        "name": "FPC_ENABLED",
+        "fromHost": true
+    },
+    {
+        "name": "RUN_CCAAS",
+        "fromHost": true
+    },
+
+    {
+        "name": "FABRIC_LOGGING_SPEC",
+        "fromHost": true
+    }
+]
+}
```

Comment:
I think this file should move to `samples/chaincode/cc-tools-demo/`

Reply:
Done. But after this way after the make for `ecc` target
```Makefile
ecc: ecc_dependencies
	ego-go build $(GOTAGS) -o $(ECC_BINARY) main.go
	cp $(EGO_CONFIG_FILE) ./enclave.json
	ego sign
	ego uniqueid $(ECC_BINARY) > mrenclave
	ego bundle $(ECC_BINARY) $(ECC_BUNDLE)
```

The user will have two versions of this file, one is name `enclave.json` and the other is `ccToolsDemoEnclave.json`.